etc-conf/rhsm.conf

```diff
@@ -71,7 +71,7 @@ pluginConfDir = /etc/rhsm/pluginconf.d
 auto_enable_yum_plugins = 1
 
 # Run the package profile on each yum/dnf transaction
-package_profile_on_trans = 0
+package_profile_after_command = 0
```

Comment:
If we are going to change the name of this parameter, please update the man pages as well.

Reply:
Done.